docs/admin/keycloak/keycloak-saml-sso.md

```diff
@@ -6,13 +6,68 @@ tags:
   - SSO
 ---
 
-## This content is in progress
+## Overview
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+The Janssen Server acts as a SAML Identity Provider to support outbound SAML
+single sign-on (SSO). It does this by leveraging the SAML features available
+in [Keycloak](https://www.keycloak.org/), while leveraging the Janssen Server's
+authentication module to handle the user
+authentication part of SSO. Once authentication is successful, the user is
+redirected to the SP with personal attributes and an active SSO session.
+
+## Installation
+
+During installation of the Janssen Server, simply select the option
+`Install Jans KC`   to install and setup SAML SSO for Keycloak. Instructions
+on how to setup SAML SSO with Keycloak post-install will eventually be provided.
+
+
+## Managing SAML Service Providers Through the Jans-Cli
+
+To act as an IDP to various SAML SPs (Service Providers), the latter
+need to be added to the Janssen Server. This can be done via the
+[Jans TUI](../config-guide/config-tools/jans-tui/README.md) which is what we
+will cover in this section.
+
+###  Adding a SAML SP
+
+- Open Jans-TUI
```

Comment:
Since these steps are ordered I would prefer if we make it an ordered list I.e number them instead

Reply:
@moabu I have added new commit to address this comment. Please check.